docs/rfc/rfc-1.adoc

```diff
@@ -504,36 +504,62 @@ this transaction.
 [[transaction-incentives]]
 === Transaction Incentives
 
-Governable Parameters:
-
+Governable Parameters
```

Comment:
```suggestion
Governable Parameters:

```
Looks that this got in accidentally...

Reply:
[753109d](https://github.com/keep-network/tbtc-v2/pull/125/commits/753109dd65daab9333599f01d8d747950547eeb6)